### PR TITLE
PR #2 for User Context Implementation - Rights Helper Interface

### PR DESCRIPTION
**Pull request by ghost**
This is no2 of the PR for multi user capabilities. This PR introduces a new Interface on the RightsHelper to clearly identify when the check for access is up to the system user or to the user context.

PR #571 needs to be merged before.

### CODE REVIEW

File: de.dlr.sc.virsat.model/src/de/dlr/sc/virsat/model/dvlm/roles/RightsHelper.java [hunk 1]
```diff
@@ -103,12 +103,8 @@ public static synchronized boolean hasWritePermission(EObject eObject, IUserCont
 	 * @param eObject The eObject to be tested for write access
 	 * @return true in case the user has access or is a super user.
 	 */
-	public static boolean hasWritePermission(EObject eObject) {
+	//public static boolean hasSystemUserWritePermission(EObject eObject) {
```
Comment (SaMuellerDLR) [inline de.dlr.sc.virsat.model/src/de/dlr/sc/virsat/model/dvlm/roles/RightsHelper.java:106]: Some leftover outcommented line.

**Review by SaMuellerDLR — CHANGES_REQUESTED** [review]
Just a very minor thing.

File: de.dlr.sc.virsat.model/src/de/dlr/sc/virsat/model/dvlm/roles/RightsHelper.java (hunk 1 above)
Comment (PhilMFischer) [reply de.dlr.sc.virsat.model/src/de/dlr/sc/virsat/model/dvlm/roles/RightsHelper.java]: Removed...

**Review by SaMuellerDLR — APPROVED** [review]
Approved!

**Review by franzTobiasDLR — APPROVED** [review]

**Pull request merged**